BOUNTY_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Bug Submission
+
+## Summary
+
+## Detailed Description
+
+## Attack Scenario
+
+## Proposed Impact
+
+## Additional Notes
```

Comment:
at the top, just add a note describing what this document is. 

This template is used for submitting bounty requests related to the AirSwap protocols repository. Email us at bounty@airswap.io with all contents of this template for a prompt response.